libbeat/processors/add_kubernetes_metadata/kubernetes.go

```diff
@@ -37,9 +40,13 @@ import (
 )
 
 const (
-	timeout = time.Second * 5
+	timeout                = time.Second * 5
+	selector               = "kubernetes"
+	checkNodeReadyAttempts = 10
 )
 
+var once sync.Once
```

Comment:
Don't use a global for this please, `add_kubernetes_metadata` could be defined in several parts of the config (or in some dynamic config), and only the first appearance would be initialized.